### PR TITLE
Update Emarsys to use public API's

### PR DESCRIPTION
**Pull request by jyucis**
Commit removes escher request signing and uses the public APIs. Sets
the gateway to production by default because sandboxes are only issues by special
request.

### CODE REVIEW

File: app/lib/workarea/emarsys/gateway.rb [hunk 1]
```diff
@@ -39,32 +32,28 @@ def connection
...
-            algo_prefix: 'EMS',
-            vendor_key: 'EMS',
-            auth_header_name: 'X-Ems-Auth',
-            date_header_name: 'X-Ems-Date',
-            current_time: Time.current
-          }
+        def auth_header
+          nonce = SecureRandom.base64(32).first(32)
+          timestamp = Time.now.utc.iso8601
+          text = nonce + timestamp + secret_key
```
Comment (JasonHummel4) [inline app/lib/workarea/emarsys/gateway.rb]: text is an unused variable

File: README.md [hunk 2]
```diff
@@ -35,17 +35,17 @@ Configuration
...
 
-1. **Production:** A toggle switch to set the integration into production mode, defaults to false.
-2. **Customer ID:** The customer ID of account. Ask your Emarsys account manager for this value.
-3. **Merchant ID:** The merchant ID of the Emarsys account. This value is required for the Web Extend Analytics Integration
+1. **Production:** A toggle switch to set the integration into production mode, defaults to true.
+2. **Merchant ID:** The merchant ID of the Emarsys account. This value is required for the Web Extend Analytics Integration
 
 
-The contact and sales data APIs require different access keys that require storage in the secrets file.
+The contact and sales data APIs require different access keys that require storage in the secrets file. You can generate a user for the contact API in the emarsys admin.
```
Comment (bencrouse) [inline README.md]:
```suggestion
The contact and sales data APIs require different access keys that require storage in the secrets file. You can generate a user for the contact API in the Emarsys admin.
```

File: lib/workarea/emarsys.rb [hunk 3]
```diff
@@ -1,13 +1,10 @@
 require 'workarea'
 require 'workarea/storefront'
 require 'workarea/admin'
-require 'workarea/google_product_feed'
```
Comment (bencrouse) [inline lib/workarea/emarsys.rb:4]: So you added this in the Gemfile but removed it here?

**Review by bencrouse — APPROVED** [review]

**Comment by bencrouse** [comment]
@JasonHummel4 this look good to you?

**Review by JasonHummel4 — APPROVED** [review]

**Pull request merged**